src/main/kotlin/blackjack/domain/Player.kt

```diff
@@ -0,0 +1,25 @@
+package blackjack.domain
+
+class Player(
+    val name: String,
+    val hand: Hand = Hand()
+) {
+
+    init {
+        require(name.isNotBlank()) { "플레이어의 이름은 공백일 수 없습니다." }
+    }
+
+    fun init(deck: Deck) {
+        val initCards = deck.init()
+        hand.init(initCards[0], initCards[1])
+    }
+
+    fun canHit(): Boolean {
+        return hand.canHit()
+    }
+
+    fun hit(deck: Deck) {
+        val card = deck.hit()
+        hand.receive(card)
+    }
```

Comment:
```suggestion
    fun hit(card: Card) {
        hand.receive(card)
    }
```

Deck이 아닌 card를 받도록 해볼까요?

Reply:
Player로부터 Deck이라는 테스트하기 어려운 랜덤한 요소를 제거하는 작업인 것 같네요!  
hit을 하면 Deck이 아닌 한 장의 Card를 받도록 수정하겠습니다!